### PR TITLE
Make the default theme configurable

### LINKED ISSUE #557: The default theme should be configurable
This shouldn't be configurable in the html?
https://github.com/camptocamp/ngeo/blob/master/contribs/gmf/src/directives/themeselector.js#L84

### PR DESCRIPTION
**Pull request by fredj**
Fixes #557

### CODE REVIEW

**Comment by fgravin** [comment]
I think we were more talking about an `angular.value` in the index.html file, generated on page loading.

See https://github.com/Geoportail-Luxembourg/geoportailv3/blob/master/geoportailv3/templates/index.html#L275

**Comment by sbrunner** [comment]
Personally I think about this ...

**Comment by fgravin** [comment]
What this ? the PR or my comment ?
Where the default theme will come from ? DB ?

**Comment by fredj** [comment]
What about adding a `defaultTheme` in the response from the 'themes' webservice?

**Comment by sbrunner** [comment]
this => the Pull request :-)

**Comment by sbrunner** [comment]
In standard case it's a configuration from the viewer.js.

**Comment by fgravin** [comment]
Ok i don't mind.

File: contribs/gmf/src/directives/themeselector.js [hunk 1]
```diff
@@ -18,6 +18,7 @@ gmf.themeselectorDirective = function() {
     restrict: 'E',
     controller: 'gmfThemeselectorController',
     scope: {
+      'defaultTheme': '@gmfThemeselectorDefaulttheme',
```
Comment (sbrunner) [inline contribs/gmf/src/directives/themeselector.js]: The `@htmlAttribute` should be added, see: https://github.com/camptocamp/ngeo/blob/master/contribs/gmf/src/directives/search.js#L47-L50

Comment (sbrunner) [inline contribs/gmf/src/directives/themeselector.js]: Then for me you can merge :-)

**Comment by sbrunner** [comment]
Thanks :-)

**Pull request merged**